tensorflow/core/kernels/depthwise_conv_op_gpu.h

```diff
@@ -1123,14 +1127,18 @@ __global__ void __launch_bounds__(640, 2)
 // Device function to compute sub-warp sum reduction for a power-of-two group of
 // neighboring threads.
 template <int kWidth, typename T>
+#if GOOGLE_CUDA
 __device__ __forceinline__ T WarpSumReduce(T val) {
```

Comment:
Is __forceinline__ not supported on ROCm? If it's not, should we introduce a preprocessor macro instead of #if-switching every occurrence.

Reply:
Good catch. I think it is available in ROCm as well, according to [here](https://github.com/ROCm-Developer-Tools/HIP/blob/master/include/hip/hcc_detail/host_defines.h#L55). Will remove the macro.